parser_library/test/expressions/ca_symbol_test.cpp

```diff
@@ -30,7 +30,7 @@ TEST(ca_symbol, undefined_attributes)
     evaluation_context eval_ctx { ctx, library_info_transitional::empty, diags };
     std::string name = "n";
 
-    ca_symbol sym(&name, range());
+    ca_symbol sym(ctx.ids().add(name), range());
```

Comment:
```suggestion
    ca_symbol sym(ctx.ids().add("n"), range());
```

and delete the `name` variable above?